package lte.rrc;

message AttachReq {
    required int32 id = 1;
}

message AttachResp {
    required int32 id = 1;
    
    enum Status {
        OK = 0;
        NOK = 1;
    }
    required Status status = 2;
}

message DetachReq {
    required int32 id = 1;
}

message DetachResp {
    required int32 id = 1;
    
    enum Status {
        OK = 0;
        NOK = 1;
    }
    required Status status = 2;
}

message RRC{
    optional AttachReq  attach_req  = 1;
    optional AttachResp attach_resp = 2;
    optional DetachReq  detach_req  = 3;
    optional DetachResp detach_resp = 4;
}